shoes-swt/lib/shoes/swt/common/focus.rb

```diff
@@ -6,6 +6,14 @@ module Focus
         def focus
           @real.set_focus
         end
+
+        def focused?
+          @real.has_focus
+        end
+
+        alias focussed? focused?
+        alias focussed focused?
+        alias focused focused?
```

Comment:
Looking it over, I'd actually be inclined to only provide the aliases on the shoes-core side, not on the GUI. I'm fine with the contract of the backends (which end-users never access directly) saying "you've got to use `focused?` as the primary one, while providing the convenience for Shoes apps themselves of the aliases. Mind making that change and then I'll get it merged?

Reply:
Ah that makes sense, will change. Thanks!

Reply:
The specs are for the gui and I don't see a spec for the shoes-core. Are the alias specs worth it enough?

Reply:
I'm fine with letting this stand if you're ready to be done with the issue.

That said, I've been slowly adding in specs for the various `common` modules that didn't get direct testing way back when they were created (or are tested indirectly through other element types). An example is `shoes-core/spec/shoes/common/visibility_spec.rb`.

I do think it's probably worthwhile to add at the shoes-core level, just to document the expected interaction between shoes-core and the backends, but it's not a huge problem to leave it like it is for now.

Reply:
I'll give it a shot!